.github/workflows/release.yml

```diff
@@ -43,7 +43,6 @@ jobs:
         with:
           commit: 'build: version packages'
           title: 'build: version packages'
-          version: npm run version
```

Comment:
For anyone else wondering, the [changeset action](https://github.com/changesets/action#inputs) defaults to essentially `npx changeset version` if we remove this input (like this PR does). So this should work.

Reply:
Indeed. Sorry, I should have left that as a comment!